plugins/server.go

```diff
@@ -68,7 +68,7 @@ func (s *Server) start() error {
 	if err != nil {
 		return fmt.Errorf("error getting environment configuration: %w", err)
 	}
-	s.PromServer = NewPromServer(envCfg.PrometheusPort(), s.lggr)
+	s.PromServer = NewPromServer(envCfg.PrometheusPort(), s.Logger)
```

Comment:
why did this use to compile?

Reply:
It was referencing a field from one of the embedded types. Maybe the very same field it was meant to be updating from prom? 🤦